lib/backburner.js

```diff
@@ -73,6 +79,48 @@ Backburner.prototype = {
     }
   },
 
+  _trigger: function(eventName /* ...args */) {
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
+      var len = arguments.length;
+      var args = new Array(len - 1);
+      for (var i = 1; i < len; i++) {
+        args[i - 1] = arguments[i]
+      }
+      for (var i = 0; i < callbacks.length; i++) {
+        callbacks[i].apply(null, args);
+      }
+    }
+  },
+
+  on: function(eventName, callback) {
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
+      callbacks.push(callback);
+    }
+  },
+
+  off: function(eventName, callback) {
+    if (eventName) {
+      var callbacks = this._eventCallbacks[eventName];
+      if (!callbacks) return;
+      if (callback) {
+        for (var i = 0; i < callbacks.length; i++) {
+          if (callbacks[i] === callback) {
+            callbacks.splice(i, 1);
+            i--;
+          }
+        }
+      } else {
+        this._eventCallbacks[eventName] = [];
```

Comment:
This allows you to remove all listeners for a specific event, but perhaps we should only allow off on specific event/function combos